cleanlab_studio/studio/studio.py

```diff
@@ -210,10 +210,18 @@ def create_project(
             api_key=self._api_key,
             dataset_id=dataset_id,
             project_name=project_name,
-            tasktype=tasktype,
+            task_type=task_type,
             modality=modality,
-            modeltype=modeltype,
+            model_type=model_type,
             label_column=label_column,
             feature_columns=feature_columns if feature_columns is not None else [],
             text_column=text_column,
         )
+
+    def get_latest_cleanset_id(self, project_id: str) -> Optional[str]:
```

Comment:
function name doesn't indicate that the function is blocking. I personally would _prefer_ splitting this up into `poll` and `get` steps (like I commented for the util functions above), but at the least, the function name needs to indicate that it's blocking.

an alternate approach here that's fairly idiomatic is to use a flag for `blocking`. if the user sets `blocking=True`, this function is blocking and waits until a cleanset ID is ready. if the user sets `blocking=False`, we return None if the cleanset ID isn't ready. would also recommend adding a timeout argument